packages/tc-ui/src/lib/components/input-wrapper.jsx

```diff
@@ -58,7 +58,7 @@ const WrappedEditComponent = props => {
 	return (
 		<WrapperTag
 			className="o-forms-field"
-			data-biz-ops-type={props.componentType}
+			data-treecreeper-component={props.componentType}
```

Comment:
- [ ] Do all components initialise ok? 
- [ ] Does MVR still work?

Reply:
Document component doesn't seem to initialise (but might've been broken already)

Reply:
Yes - document component broken on master - will raise a ticket